cmd/utils.go

```diff
@@ -130,3 +132,13 @@ func lineTrunc(content string, maxLen int) string {
 	sndPart := string(runes[rlen-halfLen:])
 	return fstPart + "…" + sndPart
 }
+
+func getLookupType(l string) minio.BucketLookupType {
+	switch l {
```

Comment:
I think it is better to lower case `l`, in case if a user puts a value manually in the config file

Reply:
that's a good idea.